beacon-chain/p2p/peers/status.go

```diff
@@ -449,6 +450,32 @@ func (p *Status) InboundConnected() []peer.ID {
 	return peers
 }
 
+// InboundConnectedTCP returns the current batch of inbound peers that are connected using TCP.
+func (p *Status) InboundConnectedTCP() []peer.ID {
```

Comment:
Instead of having separate methods for different protocols, how about:
```
func (p *Status) InboundConnected(protocol string) []peer.ID
```
that way, you don't have to duplicate the implementation.

Reply:
Fixed in https://github.com/prysmaticlabs/prysm/pull/13786/commits/f42455b6acf2323e38729bd3979cf60cd283ea0f.